.github/ISSUE_TEMPLATE/docDB.yml

```diff
@@ -0,0 +1,14 @@
+name: Core-DocDB
```

Comment:
if you will add project, this is the main one: https://github.com/yugabyte/yugabyte-db/projects/6

Reply:
Thank you! At the moment since Github does not support adding Projects to Issue forms, its on hold. Hopefully , it's  available in the near future.